### PR TITLE
AA-13147: Add country filter documentation

### PR DESCRIPTION
**Pull request by AlithAnar**
# Deploy preview

For the time being, deploy previews are generated using the Nelify CLI. More info on Slack (#plaform-docs).

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/AA-13147)

# Description

Docs for a new `countries` filter for the `reports-api` and archives v3.5

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/data-reporting/reports-api/v3.5/index.mdx [hunk 1]
```diff
@@ -76,14 +76,15 @@ You can authorize your calls to the Reports API using one of the following metho
...
 | `filters.agent_response.agents.<string_filter_type>`<sup>**2**</sup>                       | No                  | `any`     | Described below. Works only if the `agent_response.first` filter is set to `true`.                                                                                                                                                          |
 | `filters.agent_response.groups.<integer_filter_type>`<sup>**3**</sup>                      | No                  | `any`     | Described below. Works only if the `agent_response.first` filter is set to `true`.                                                                                                                                                          |
+| `filters.customer_countries.<string_filter_type>`<sup>**2**</sup>                          | No                  | `any`     | Described below. It supports country codes with `ISO 3166-1 Alpha-2` format.                                                                                                                                                                |
```
Comment (ale-kacperczyk) [inline src/pages/data-reporting/reports-api/v3.5/index.mdx:87]:
```suggestion
| `filters.customer_countries.<string_filter_type>`<sup>**2**</sup>                          | No                  | `any`     | Described below. It supports country codes with the `ISO 3166-1 Alpha-2` format.                                                                                                                                                                |
```

Comment (ale-kacperczyk) [reply src/pages/data-reporting/reports-api/v3.5/index.mdx:87]: Please apply the same change in the docs below.

**Review by ale-kacperczyk — COMMENTED** [review]
This i a new functionality so it should land in Changelogs

**Comment by ale-kacperczyk** [comment]
Preview: https://624ea1ace9c25f4a83116920--livechat-public-docs.netlify.app

**Review by ale-kacperczyk — COMMENTED** [review]
Could you rebase your branch, please?

File: src/pages/messaging/agent-chat-api/changelog/index.mdx [hunk 2]
```diff
@@ -21,6 +21,11 @@ The developer preview version provides a preview of the upcoming changes to the
 
 ## [v3.5] - Developer preview
 
+### Chats
+
+- The **List Archives** ([Web](/messaging/agent-chat-api/v3.5/#list-archives) & [RTM](/messaging/agent-chat-api/v3.5/rtm-reference/#list-archives)) method:
```
[the remark this answers is not in the document]
Comment (AlithAnar) [reply src/pages/messaging/agent-chat-api/changelog/index.mdx]: fixed in https://github.com/livechat/livechat-public-docs/pull/1113/commits/5ff3157f3bbbfcb3d301231a2014874e27e0dca0

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**